packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -256,6 +262,10 @@ const styles = StyleSheet.create({
         flexDirection: "row",
         padding: Spacing.xSmall_8,
     },
+    containerFloating: {
+        padding: Spacing.xSmall_8,
```

Comment:
I opted to use padding instead of margins because it was much more consistent with the spacing. Margins kept collapsing.